dev/README.md

```diff
@@ -9,11 +9,11 @@ for packages. It follows format:
     "builders":[
         {
             "name":"xxx",
-            "repo":"packages"
+            "repo":"packages",
```

Comment:
probably change name to yyy?

Reply:
Here only one builder is listed in the doc. I guess we can use xxx.